src/RageSoundManager.cpp

```diff
@@ -133,9 +133,9 @@ float RageSoundManager::GetPlayLatency() const
 int RageSoundManager::GetDriverSampleRate() const
 {
 	if( m_pDriver == nullptr )
-		return 44100;
+		return kFallbackSampleRate;
 
-	return m_pDriver->GetSampleRate();
+	return m_pDriver->GetSampleRate(); // Returns the *actual* operating rate of the loaded driver
```

Comment:
Here's an generally throughout, let's put the comment above instead of on the same line (since it'll be safer for the line length limit)